justfile

```diff
@@ -191,6 +191,11 @@ create platform=default_platform:
         "AKS-PEER-SNP")
             just upload-image
 
+            # Populate Terraform variables.
+            echo "name_prefix = \"$azure_resource_group\"" > infra/azure-peerpods/just.auto.tfvars
```

Comment:
Maybe we can add a suffix here so this won't collide with the rg used for AKS?

Reply:
We already add a suffix in our Terraform configs, so that'd be a bit redundant:
https://github.com/edgelesssys/contrast/blob/0662a2ee4b80bf22441b5a828d2e0f3a6b598320/infra/azure-peerpods/main.tf#L36
Do you still want me to add a suffix?